### PR TITLE
Add recipe for itk

### PR DESCRIPTION
**Pull request by thewtex**
This re-uses the PyPI packages. The infrastructure is based off the tensorflow
conda recipe.

Currently a monolithic package for "itk" in created. In the future, it is
possible to splite it into an itk metapackage and multiple dependent packages,
itk-core, itk-io, itk-segmentation, etc.

Co-authored-by: Eric Dill <thedizzle@gmail.com>

CC: @ericdill @jjhelmus @blowekamp @msarahan

### CODE REVIEW

**Comment by thewtex** [comment]
Todos:

- [ ] Develop / test ``bld.bat` for Windows
- [ ] Rename current `itk` package to `libitk`

File: recipes/itk/bld.bat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+%PYTHON% -m pip install --no-deps https://pypi.org/packages/cp35/t/tensorflow/tensorflow-%PKG_VERSION%-cp35-cp35m-win_amd64.whl
```
Comment (ericdill) [inline recipes/itk/bld.bat]: @thewtex I think you need to do the same thing here that we did for build.sh. But I have no idea how to do loops in windows batch scripts 🤷‍♂️

File: recipes/itk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    - numpy
+
+test:
+  imports:
+    - itk
+
+about:
+  home: http://www.itk.org/
+  license: Apache 2.0
+  summary: ITK is an open-source toolkit for multidimensional image analysis
```
Comment (ericdill) [inline recipes/itk/meta.yaml]: might want to include

```yaml
about:
  doc_url: https://itk.org/ITK/help/documentation.html
  dev_url: https://github.com/InsightSoftwareConsortium/ITK
```

File: recipes/itk/bld.bat (hunk 1 above)
Comment (thewtex) [inline recipes/itk/bld.bat]: Will push up the addition in the next version :-) :tada:

File: recipes/itk/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    - python
+    - numpy
+
+test:
+  imports:
+    - itk
+
+about:
+  home: http://www.itk.org/
+  license: Apache 2.0
```
Comment (thewtex) [inline recipes/itk/meta.yaml]: Good point. A quick search, it looks like it does not get packed, though.

File: recipes/itk/meta.yaml (hunk 2 above)
Comment (thewtex) [inline recipes/itk/meta.yaml]: :+1: Excellent, will do.

**Comment by thewtex** [comment]
@ericdill Thanks for the great review! :1st_place_medal: 

Locally built and testing with miniconda3 / Python 3.6 on Linux / macOS / Windows.

**Comment by jjhelmus** [comment]
The Mac OS X build is failing only because rustc is being built.  All three itk builds succeed.  I don't see any reason why this should be a blocker for merging since it is unrelated to the current recipe.

**Comment by jjhelmus** [comment]
Shoot, this is probably going to cause issues because of the existing itk-feedstock, conda-forge/itk-feedstock#8.  Sorry, I'm catching up with my emails in reverse order and this one came first.  Ping me if there is anything I can do to help fix this.

**Comment by thewtex** [comment]
@jjhelmus Regardless, for moving it forward. Could the the `conda-forge/itk-feedstock` repository please be renamed to `conda-forge/libitk-feedstock`?

**Comment by jjhelmus** [comment]
Renamed

**Comment by thewtex** [comment]
Thanks!

Should we re-merge this branch to re-populate the `itk-feedstock` repository, or does in need to be created manually?

**Pull request merged**